src/app/filecoin-plus/allocators/data/allocatorsTableColumnsData.tsx

```diff
@@ -63,3 +72,7 @@ export const allocatorsTableColumnsData = [
     },
   }),
 ]
+
+function formatDatacap(value?: string) {
+  return value ? prettyBytes(Number(value)) : 'No data available'
+}
```

Comment:
Shall we move this out to `utils`?

Reply:
Asked @filipagr and crew to clarify `No data available` [>>](https://filecoinfoundation.slack.com/archives/C06MTEQ3SP6/p1731241180071039)

Reply:
Done. Still pending response from Filipa and crew.

Reply:
As just discussed - We figured out this is about inconsistencies between addresses in the two different datasets. Will follow up with the Filecoin Plus team.